module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -169,6 +169,51 @@ Read the full documentation and download cheatsheets (where available) for these
 
 ### `Seurat` and `SCE` object conversion
 
+This [vignette](https://satijalab.org/seurat/articles/conversion_vignette.html) from the `Seurat` documentation explains how to convert between `SCE` and `Seurat` formats.
```

Comment:
Rewriting for a longer, more informative link. 

```suggestion
The `Seurat` documentation provides a [vignette about converting objects](https://satijalab.org/seurat/articles/conversion_vignette.html) between `SCE` and `Seurat` formats.
```

I think it is right to link to this, but boy is it sparse. And boy do I not love instructions to install a package from the `develop` branch.

Reply:
> And boy do I not love instructions to install a package from the `develop` branch.

I bet you'll love even more how there is no `develop` branch anymore in that repo, and there are no instructions for installing the other `Seurat` package dependency which lives in a _different_ account from that one IIRC.

So, yeah, this "vignette" has problems, and I'm not at all opposed to removing it.